packages/expo-permissions/src/Permissions.types.ts

```diff
@@ -26,7 +28,7 @@ export interface PermissionMap {
 }
 
 export interface PermissionInfo extends UMPermissionResponse {
-  // iOS only - Permission.CAMERA_ROLL
+  // iOS only - Permission.MEDIA_LIBRARY/MEDIA_LIBRARY_WRITE_ONLY
```

Comment:
```suggestion
  /**
   * iOS only - Permission.MEDIA_LIBRARY/MEDIA_LIBRARY_WRITE_ONLY
   **/
```
~I'm not sure~ if non-block comments are applied to the signatures/variables and presented in VSCode 🤔 
_edit_: I've just confirmed that they're not 😉